bundles/com.salesforce.bazel.eclipse.core/src/com/salesforce/bazel/eclipse/core/classpath/BazelClasspathContainer.java

```diff
@@ -49,14 +50,16 @@
  */
 public class BazelClasspathContainer implements IClasspathContainer, Serializable {
 
-    private static final long serialVersionUID = 390898179243551621L;
+    private static final long serialVersionUID = 390898179243551622L;
 
     private final IPath path;
     private final IClasspathEntry[] classpath;
+    private final IClasspathEntry[] unloadedClasspath;
 
-    public BazelClasspathContainer(IPath path, IClasspathEntry[] classpath) {
+    public BazelClasspathContainer(IPath path, IClasspathEntry[] classpath, IClasspathEntry[] unloadedClasspath) {
         this.path = path;
         this.classpath = classpath;
+        this.unloadedClasspath = unloadedClasspath;
```

Comment:
```suggestion
        this.classpath = requireNotNull(classpath);
        this.unloadedClasspath = requireNotNull(unloadedClasspath);
```

Can you wrap this into a `requireNotNull` (needs static import from Objects)?